make/Makefile.common

```diff
@@ -49,13 +49,13 @@ endif # !os_name
 
 cfg ?= release
 
-device_type ?= GPU
+device_type ?= level_zero:gpu
```

Comment:
Does the changes fin this file really linked with this PR ?

Reply:
It is a fix for `Makefile.common`. We can separate that change from this PR.

Reply:
https://github.com/uxlfoundation/oneDPL/pull/2009